### DISCUSSION TITLE
[Enhancement] [config-provider] Config Provider对于组件的支持

### DISCUSSION THREAD

**Discussion by 2292777095** [element-plus/element-plus#13555] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `config-provider`

## Description

目前文档上Config Provider只支持message和button组件，对于其他组件是否支持，如果支持请更新文档，如果不支持请添加，否则el-config-provider组件的功能就很鸡肋，例如dialog、card、select、form等等一些高频使用的组件可以优先添加

### Additional comments

_(empty)_

**Comment by xxholly32** [comment]
https://github.com/xxholly32/vite-plugin-custom-attr

如果你用vite的话，可以考虑下这个插件，可以解耦大部分的场景

**Comment by 2292777095** [comment]
vite-plugin-custom-attr这个插件有问题，配合element-plus，在template中写箭头函数会报错，[vite] Internal server error: Whitespace was expected.






------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年7月14日(星期五) 中午11:46
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [element-plus/element-plus] [Enhancement] [config-provider] Config Provider对于组件的支持 (Discussion #13555)





 
https://github.com/xxholly32/vite-plugin-custom-attr
 
如果你用vite的话，可以考虑下这个插件，可以解耦大部分的场景

**Comment by 2292777095** [comment]
vite-plugin-custom-attr这个插件有问题，配合element-plus，在template中写箭头函数会报错，[vite] Internal server error: Whitespace was expected.


测试后发现，如果在配置文件中配置了el-button标签的属性，那么template中el-button就不能使用箭头函数，否则就会报上面的错误


------------------ 原始邮件 ------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年7月14日(星期五) 中午11:46
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [element-plus/element-plus] [Enhancement] [config-provider] Config Provider对于组件的支持 (Discussion #13555)





 
https://github.com/xxholly32/vite-plugin-custom-attr
 
如果你用vite的话，可以考虑下这个插件，可以解耦大部分的场景
